tests/bears/BearTest.py

```diff
@@ -515,7 +515,7 @@ def setUp(self):
         super().setUp()
         self.mock_url = 'https://test.com'
         self.filename = 'test.html'
-        self.teapot_url = 'https://httpstat.us/418'
+        self.teapot_url = 'https://httpbin.org/status/418'
```

Comment:
I think we should use the Google Teapot.

See https://github.com/coala/coala/issues/5628#issuecomment-404678858 for background.